<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmframe</class>
 <widget class="QWidget" name="frmframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Choose Sampling Frame:</string>
         </property>
         <property name="buddy">
          <cstring>tableView</cstring>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select frame that describes physically and temporally  the process of data collection if already exists, otherwise create new one!</string>
       </property>
       <property name="statusTip">
        <string>Select frame that describes physically and temporally  the process of data collection if already exists, otherwise create new one!</string>
       </property>
       <property name="whatsThis">
        <string>Select frame that describes physically and temporally  the process of data collection if already exists, otherwise create new one!</string>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushNew">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Create new record</string>
         </property>
         <property name="statusTip">
          <string>Create new record</string>
         </property>
         <property name="whatsThis">
          <string>Creates new record</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/add.png</normaloff>:/app_new/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Edit record</string>
         </property>
         <property name="statusTip">
          <string>Edit record</string>
         </property>
         <property name="whatsThis">
          <string>Edits record</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/pencil.png</normaloff>:/app_new/pencil.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushRemove">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Remove record</string>
         </property>
         <property name="statusTip">
          <string>Remove record</string>
         </property>
         <property name="whatsThis">
          <string>Removes record</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/cancel.png</normaloff>:/app_new/cancel.png</iconset>
         </property>
         <property name="shortcut">
          <string>Del</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDetails">
     <property name="toolTip">
      <string>Input data to physically and temporally describe the process of data collection;</string>
     </property>
     <property name="statusTip">
      <string>Input data to physically and temporally describe the process of data collection;</string>
     </property>
     <property name="whatsThis">
      <string>Input data to physically and temporally describe the process of data collection;</string>
     </property>
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QGroupBox" name="groupPhysical">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Describe physical characteristics of the frame;</string>
        </property>
        <property name="statusTip">
         <string>Describe physical characteristics of the frame;</string>
        </property>
        <property name="whatsThis">
         <string>Describe physical characteristics of the frame;</string>
        </property>
        <property name="title">
         <string>Physical Frame</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Choose Frame</string>
           </property>
           <property name="buddy">
            <cstring>cmbPrexistent</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbPrexistent">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="toolTip">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="toolTip">
            <string>View Frame</string>
           </property>
           <property name="statusTip">
            <string>View Frame</string>
           </property>
           <property name="whatsThis">
            <string>Allows viewing this frame</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbSource">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(220, 220, 220, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
font: 75 10pt &quot;Fixedsys&quot;;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>[SOURCE]</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupTime">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="title">
         <string>Time Frame</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Start Date</string>
           </property>
           <property name="buddy">
            <cstring>customDtStart</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="customDtEnd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumDate">
            <date>
             <year>2070</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1969</year>
             <month>12</month>
             <day>23</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDateEdit" name="customDtStart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumDate">
            <date>
             <year>2070</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1969</year>
             <month>12</month>
             <day>23</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>End Date</string>
           </property>
           <property name="buddy">
            <cstring>customDtEnd</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbHeader">
        <property name="text">
         <string>LBHEADER</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QGroupBox" name="groupProcess">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Sampling Technique</string>
           </property>
           <property name="buddy">
            <cstring>toolProcess</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolProcess">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="toolTip">
            <string>Characterise the sampling methodology(ies);</string>
           </property>
           <property name="statusTip">
            <string>Characterise the sampling methodology(ies);</string>
           </property>
           <property name="whatsThis">
            <string>Characterise the sampling methodology(ies);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBackNext">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushPrevious">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="icon">
         <iconset resource="app_new.qrc">
          <normaloff>:/app_new/back.png</normaloff>:/app_new/back.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushNext">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="app_new.qrc">
          <normaloff>:/app_new/forward.png</normaloff>:/app_new/forward.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>pushNew</tabstop>
  <tabstop>pushEdit</tabstop>
  <tabstop>pushRemove</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>pushPrevious</tabstop>
  <tabstop>pushNext</tabstop>
 </tabstops>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushNext</sender>
   <signal>clicked()</signal>
   <receiver>frmframe</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushNew</sender>
   <signal>clicked()</signal>
   <receiver>frmframe</receiver>
   <slot>createRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushEdit</sender>
   <signal>clicked(bool)</signal>
   <receiver>frmframe</receiver>
   <slot>editRecord(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRemove</sender>
   <signal>clicked()</signal>
   <receiver>frmframe</receiver>
   <slot>removeRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbPrexistent</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>frmframe</receiver>
   <slot>adjustFrmSamplingMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>frmframe</receiver>
   <slot>previewRow(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>frmframe</receiver>
   <slot>onButtonClick(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbPrexistent</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>frmframe</receiver>
   <slot>onFrameChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChoosePreExistent()</slot>
  <slot>onCreateNew()</slot>
  <slot>next()</slot>
  <slot>apply()</slot>
  <slot>createRecord()</slot>
  <slot>onButtonClick(QAbstractButton*)</slot>
  <slot>previewRow(QModelIndex)</slot>
  <slot>editRecord(bool)</slot>
  <slot>removeRecord()</slot>
  <slot>updateSample()</slot>
  <slot>adjustFrmSamplingMode()</slot>
  <slot>onFrameChange(int)</slot>
 </slots>
</ui>
